### DISCUSSION TITLE
UP主个人主页报错

### DISCUSSION THREAD

**Discussion by twtan** [ywmoyue/biliuwp-lite#846] [Q&A] [answered]
### Bug 描述

点开视频UP主的主页的时候会提示“读取个人资料失败”，几乎每次都会报这个错误

### 操作步骤

```plain text
点开视频，点击UP主信息，报错
```

### 期望结果

可以正常使用

### 截图

![image](https://github.com/user-attachments/assets/0db0d1ea-0ead-4575-875f-2e1c243a74a5)

### 日志

2024-10-02 16:58:01.9760|ERROR|3|UserDetailViewModel.GetUserInfo|读取个人资料失败|风控校验失败
   at BiliLite.ViewModels.User.UserDetailViewModel.GetUserInfoCore()
   at BiliLite.ViewModels.User.UserDetailViewModel.GetUserInfo()

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment, marked as answer]
这个问题在4.7.3修复过一次，请问你这边现在是什么版本

**Comment by twtan** [comment]
> 这个问题在4.7.3修复过一次，请问你这边现在是什么版本

看了一下我还是4.7.2 ！
现在最新都4.7.4...居然都没提示更新...

更新4.7.4后没报错了

**Comment by ywmoyue** [comment]
要提示信息需要访问github获取版本信息文件，可能网络问题导致提示没有吧

**Comment by ywmoyue** [comment]
问题重新出现了，我看看怎么处理
